File: ServiceDemo/src/main/java/com/servicedemo/server/BinderBImpl.java
package com.servicedemo.server;

import android.os.RemoteException;
import android.util.Log;

public class BinderBImpl extends IBinderB.Stub{
    @Override
    public void methodB(String paramB) throws RemoteException {
        Log.e("lzw","methodB");
    }
}

File: ServiceDemo/src/main/java/com/servicedemo/server/BinderBImpl.kt
package com.servicedemo.server

import android.os.RemoteException
import android.util.Log
import com.servicedemo.server.IBinderB.Stub

class BinderBImpl : Stub() {
    @Throws(RemoteException::class)
    override fun methodB(paramB: String) {
        Log.e("lzw", "methodB")
    }
}